### DISCUSSION TITLE
iOS: wrong Wikipedia language

### DISCUSSION THREAD

**Discussion by Tom7320** [osmandapp/OsmAnd#11736] [Q&A]
Hi!

Maybe this is a very basic beginner's question, and I am just to stupid to find the answer myself. So please be patient with me... :wink:

I installed OsmAnd on my iPhone and also downloaded Wikipedia articles for my region. But it seems that OsmAnd downloaded the wrong language?!? How do I correct that?

Thx a lot!

Thorsten

**Comment by vshcherb** [comment]
Wikipedia Files are not distributed by language, so 1 file has all languages inside.

**Reply by Tom7320** [reply]
OK, thanks! But how do I change the language? It's definitely the wrong language...

**Reply by barteis** [reply]
Assuming you're looking for English, are you sure an English language Wikipedia article of the feature of interest exists? I've seen cases in the U.S.  where the entry for a local feature doesn't appear in the English Wikipedia.  Yet it does in another language.

**Reply by Tom7320** [reply]
Well I was looking for a (definitely existing) article about a village in Germany in German language...
Is there a way I can choose/select what language to prefer if there are more than one language available for a certain topic?

**Reply by barteis** [reply]
On the Android version, yes.  On the iOS version, I don't think so.

**Reply by dmpr0** [reply]
Hello, @Tom7320!
You can change the language for an opened article in the top right corner. Also, you can open Search, and type "wiki" where you can select the needed language.
![photo_2021-05-26 16 24 40](https://user-images.githubusercontent.com/9365308/119667357-e2f6b180-be3e-11eb-83d7-f1330f8a8459.jpeg)

**Comment by Tom7320** [comment]
![IMG_1269](https://user-images.githubusercontent.com/19165864/119766843-0741a500-beb6-11eb-80c4-f9d7dc8faa58.PNG)

I don't see a way to change the language... :thinking:

**Reply by vshcherb** [reply]
You need to click on article and inside you can see the list of languages. Wikipedia layer and instruments will be improved.
